server/index.js

```diff
@@ -20,11 +31,25 @@ app.prepare().then(() => {
       res.send(JSON.stringify(files));
     });
   });
+
+  server.get('/_api/content/getMedia/:name', (req, res) => {
```

Comment:
`:name` kinda reminds me of a person related field. Let's rename to `filename` for now. would be to use great to use uuid's eventually, if we implement storing media meta data in json